.circleci/config.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           # of a package name and a version specifier"
           command: |
             pip install pip-tools --constraint requirements.in
-            pip-compile --quiet --generate-hashes --unsafe-package=
+            pip-compile --allow-unsafe --generate-hashes --quiet
```

Comment:
`--allow-unsafe` was broken in `pip-tools==6.12.0`, but has been fixed in `pip-tools==6.12.2`